phpcs-sniffs/Formidable/Sniffs/Security/AddDirectFileAccessCheckSniff.php

```
@@ -132,22 +125,67 @@ private function hasAbspathCheck( File $phpcsFile ) {
+	}
+
+	/**
+	 * Add the ABSPATH check to a file that starts with PHP.
+	 *
+	 * @param File $phpcsFile The file being scanned.
+	 * @param int  $stackPtr  The position of the T_OPEN_TAG token.
+	 *
+	 * @return void
+	 */
+	private function addAbspathCheckToPhpFile( File $phpcsFile, $stackPtr ) {
+		$tokens = $phpcsFile->getTokens();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tokens'. (undefined)

(UnusedLocalVariable)